src/Dropdown/index.tsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react'
+import Select, { StylesConfig } from 'react-select'
+import styled from 'styled-components'
+
+const Wrapper = styled('div')`
+  ${({ width }: { width?: string }) => `
+    width: ${width || '50%'}
+  `}
+`
+
+const ValuesContainer = styled('div')`
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+`
+
+const Value = styled('div')`
+  padding: 0.5rem;
+  padding-left: 0.75rem;
+  margin: 0.5rem;
+  font-size: 1rem;
+  border-radius: 10px 0px 0px 10px;
+  color: ${(props: any) => props.textColor || '#216968'};
+  background-color: ${(props) => props.color || 'rgba(247, 173, 46, 0.6)'};
+  user-select: none;
+`
+
+const X = styled('button')`
+  all: unset;
+  margin-left: 1.3rem;
+  color: black;
+  transition: fill 0.15s ease-in-out;
+  cursor: pointer;
+  &:hover {
+    color: #bb392d;
+  }
+  &:focus {
+    color: #c82f21;
+  }
+`
+
+// TODO create types.d.ts
+interface Props {
+  options?: Array<any>
+  styles?: StylesConfig
+  selected?: any
+  isMulti: boolean
+  update: (val: any[]) => void
+}
+
+interface IDropdown {
+  (args: Props): React.ReactElement
+}
+
+const MultiSelect = (props: any) => {
+  const { value, onChange } = props
+
+  const handleRemoveValue = (e: any) => {
+    if (!onChange) return null
+    const { name: buttonName } = e.currentTarget
+    const removedValue = value.find((val: any) => val.value === buttonName)
+
+    if (!removedValue) return null
+    props.update(value)
+
+    onChange(
+      value.filter((val: any) => val.value !== buttonName),
+      { name, action: 'remove-value', removedValue }
+    )
+  }
+
+  return (
+    <Wrapper>
+      <Select
```

Comment:
is there a way of having placeholder text llike 'Select range of maturity'?
<img width="286" alt="image" src="https://user-images.githubusercontent.com/40722025/208468503-d336f9e7-3745-4430-bd45-b4a2a64c1124.png">

Reply:
yes, this is something that cna be passed as property also sorting now adding a test and updating props type

Reply:
added a test and story update